Encoder.js

```diff
@@ -1,178 +1,331 @@
 // This file includes code which was modified from https://github.com/openai/gpt-2
-const fs = require('fs')
-const path = require('path');
+// const fs = require('fs')
+// const path = require('path');
+// const json-loder
+// const loader = require("json-loader");
 
-const encoder = JSON.parse(fs.readFileSync(path.join(__dirname, './encoder.json')));
-const bpe_file = fs.readFileSync(path.join(__dirname, './vocab.bpe'), 'utf-8');
+// const encoder = loader('./encoder.json');
+
```

Comment:
We could erase these comments but in order to support loading into a browser module i have loaded the data into `encoder.js` and `bpe_ranks.js`

These are pre-computed a build time and this removes some of the work the client has to do other then load the data into memory.  Todo maybe we should clean up these comments. but theoretical you could switch it back if you only wanted nodejs.